openfisca_us/variables/gov/states/md/tax/income/credits/ctc/md_ctc.py

```diff
@@ -0,0 +1,28 @@
+from openfisca_us.model_api import *
+
+
+class md_ctc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "MD CTC"
+    definition_period = YEAR
+    unit = USD
+    documentation = "Maryland Child Tax Credit"
+
+    def formula(tax_unit, period, parameters):
+        eligible = (
+            tax_unit("adjusted_gross_income", period)
+            <= parameters(period).gov.states.md.tax.income.credits.ctc.agi_cap
+        )
+        person = tax_unit.members
+        is_dependent_and_disabled = (
+            person("is_tax_unit_dependent", period)
+            * person("is_disabled", period)
```

Comment:
We generally do:
* bool & bool
* bool * number
* number * number
```suggestion
            & person("is_disabled", period)
```